lib/update/phase_coredns.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2018 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package update
+
+import (
+	"context"
+
+	"github.com/gravitational/gravity/lib/storage"
+	"github.com/gravitational/rigging"
```

Comment:
nit: missing empty line to separate local imports from 3rd party

Reply:
I'm going to ask this every time... why do we care how automatically generated code get's generated. I get that someone has written a style guideline for this, I just don't understand why we care if goimports doesn't.

Reply:
Hmm, I don't know if that's automatically generated code. I guess you mean the difference between running goimports on a file w/o imports vs running it on a file with imports in some (alternative - i.e. grouped) format?
I personally like the grouping, but it's, well, a personal opinion. Besides, there was a precedent for goimports to adopt a custom grouping with the `-local` flag: https://github.com/golang/go/issues/12420, which will probably end this debate for both sides.
I don't feel too strong about caring whether we continue to group the imports as we do now (manually), so it's a big deal for me.